extensions/wikia/ARecoveryEngine/ARecoveryUnlockCSS.class.php

```diff
@@ -5,7 +5,7 @@ class ARecoveryUnlockCSS {
 	const API_ENDPOINT = 'sp_create_csurl';
 	const CACHE_TTL = 3600 * 10; //10h
 	const TIMEOUT = 10;
-	const WIKIA_PROXY_ENDPOINT = '/__are';
+	const WIKIA_PROXY_ENDPOINT = '/__bre';
```

Comment:
Can we reuse existing `ResourceLoaderAdEngineSourcePointCSDelivery::CS_ENDPOINT` instead of creating new const?

On the other hand most likely UnlockCSS module is going to be removed.